cmd/uhc/cluster/describe/describe.go

```diff
@@ -29,23 +30,45 @@ import (
 
 var args struct {
 	debug bool
+	out   string
+	short bool
 }
 
 var Cmd = &cobra.Command{
-	Use:   "describe CLUSTERID",
+	Use:   "describe CLUSTERID [--out] [--short]",
 	Short: "Decribe a cluster",
 	Long:  "Get info about a cluster identified by its cluster ID",
 	Run:   run,
 }
 
+// Check error
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
 func init() {
+	// Add flags to rootCmd
 	flags := Cmd.Flags()
 	flags.BoolVar(
 		&args.debug,
 		"debug",
 		false,
 		"Enable debug mode.",
 	)
+	flags.StringVar(
+		&args.out,
+		"out",
```

Comment:
Not sure if this is needed, you can just redirect the output to a file. If you still want to have the option, can it be `--output`?